### ISSUE TITLE
THEME env variable

### ISSUE THREAD

**Issue by oamyoamy** [nightscout/cgm-remote-monitor#672]
Previously, I've just manually set my theme through the browser settings. I went ahead and set the variable this time for testing... and I am not seeing it function as expected. The default (gray) is showing up in every new browser tested despite my Heroku variable being set as THEME with value colors as described here:

THEME (default) - possible values default or colors

Tested on iOS (Safari) and on desktop (Opera, Firefox, and Chrome). First load of site in ALL of them pulls up in default, regardless of the environment variable.  (Once set again via browser, cookie holds until history is cleared, and then load is default/gray again.)

**Comment by jasoncalabrese** [comment]
You browser is probably using the settings, try clicking the "Reset, and use defaults" link

<img width="298" alt="screen shot 2015-07-03 at 9 39 46 am" src="https://cloud.githubusercontent.com/assets/751143/8502638/85f91326-2167-11e5-855b-3fe9f2b9a5bc.png">

**Comment by oamyoamy** [comment]
Still didn't work. Other env setting (like SHOW_RAWBG and CUSTOM_TITLE) are holding/working. Just not THEME.

**Comment by jasoncalabrese** [comment]
What do the results for `/api/v1/status.json` look like?  Does it show `"theme":"colors"`?

**Comment by oamyoamy** [comment]
Hmmm. No... it does not. It shows "default" --> I've resaved the env variables in Heroku twice this morning in testing just to make sure it was live. 

In testing it again just now, Heroku wasn't showing the same (remove) symbol next to it either when the vars were in edit mode. 

I added the var in again several times... same icon kept showing up, and it's not immediately/easily deletable in the same way the other variables can be removed. However, in testing, I did two things... 1) I removed two other variables, taking the count from 20 to 18, and 2) I removed SHOW_RAWBG and readded it, to see if I could kick THEME up. SHOW_RAWBG automatically moved up above THEME again. But theme setting is now showing in the status.json and working.)

Unclear why it wasn't kicking in, but seems to be a Heroku issue that resolved, not a NS issue.
